js/controller.js

```diff
@@ -897,6 +897,10 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
      * @param currentTrack {{id: String, label: String, lang: String}} - current active audio track
      */
     setCurrentAudio: function(currentTrack) {
+      // user selected track should be saved to localstorage
+      var currentAudioTrack = JSON.stringify(currentTrack);
+      OO.setItem(OO.CONSTANTS.SELECTED_AUDIO, currentAudioTrack);
```

Comment:
Where is the stored value used? Just out of curiosity

Reply:
@aeng7 Thanks for the review. 
We use it later in the core to restore the default track based on user previous selection.